src/geometry/poly3/fromPoints.js

```diff
@@ -0,0 +1,27 @@
+const vec3 = require('../../math/vec3')
+
+const create = require('./create')
+
+/**
+ * Create a polygon from the given points.
+ *
+ * @param {Array[]} points - list of points
+ *
+ * @example
+ * const points = [
+ *   [0,  0, 0],
+ *   [0, 10, 0],
+ *   [0, 10, 10]
+ * ]
+ * const polygon = fromPoints(points)
+ */
+const fromPoints = (points, planeof) => {
+  if (planeof) throw new Error('use fromPointAndPlane')
```

Comment:
nice throwings !

Reply:
This is for development... I want to catch all invalid useage of this function.